### PR TITLE
Fix the tooltips of the social media icons in the navbar

### LINKED ISSUE #1389: The tooltips of the navbar social media icons are not working
**Describe the bug**
Currently, the tooltips of the navbar social media icons are not working

**To Reproduce**
1. Go to 'sef-site'
4. See error in the navbar by hovering the social media icons

**Expected behavior**
Make the tooltips working nicely in the social media icons of the navbar

**Screenshots**
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1389 

## Goals

Fixing the tooltips of the social media icons in the navbar

## Approach

Fixed the tooltips of the social media icons in the navbar

### Screenshots

image

  
### Preview Link

https://pr-1392-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: assets/content/static/navbar.html [hunk 1]
```diff
@@ -152,31 +152,31 @@ <h6 class="text-primary mb-md-1">GitHub Leaderboard</h6>
...
+                       data-toggle="tooltip" title="LinkedIn">
                         <i class="fab fa-linkedin"></i>
                         <span class="nav-link-inner--text d-lg-none">LinkedIn</span>
                     </a>
                 </li>
 
                 <li class="nav-item">
                     <a class="nav-link nav-link-icon" href="https://www.instagram.com/sefglobal/" target="_blank"
-                       data-toggle="tooltip" title="" data-original-title="Follow us on Instagram">
+                       data-toggle="tooltip" title="Instagram">
```
Comment (anjula-sack) [inline assets/content/static/navbar.html:179]: Let's use `Follow us on Instagram` as the title. Do this for the others as well

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: assets/content/static/navbar.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply assets/content/static/navbar.html:179]: So @anjula-sack, @jayasanka-sack told in the footer use only the name. So if it is ok with the follow us on Instagram shall I change it in the footer also.

Comment (YoshithaRathnayake) [reply assets/content/static/navbar.html:179]: https://github.com/sef-global/sef-site/pull/1366#discussion_r1009470793 so to me its good with only the name so what do you think @anjula-sack . What is better @anjula-sack

Comment (anjula-sack) [reply assets/content/static/navbar.html:179]: ok then let's keep it this way

**Review by anjula-sack — APPROVED** [review]
Thank you! @YoshithaRathnayake

**Pull request merged**